_data/banking.yml

```diff
@@ -42,7 +42,9 @@ websites:
       twitter: AltraCU
       facebook: altra.org
```

Comment:
```suggestion
```
twitter and facebook are not necessary if `tfa:yes`

Reply:
Can I resolve this then?